codalab/lib/bundle_cli.py

```diff
@@ -1297,7 +1266,7 @@ def do_edit_image_command(self, args):
 
         print >>self.stdout, '===='
         print >>self.stdout, 'Entering container {}'.format(container_id[:8])
-        print >>self.stdout, 'Once you are happy with the changes, please exit the container (ctrl-D)'
+        print >>self.stdout, 'If you would like to commit the changes, please exit the container (ctrl-D)'
         print >>self.stdout, 'and commit your changes to a new image by running:'
         print >>self.stdout, ''
         print >>self.stdout, '\tcl commit-image {} [image-tag]'.format(container_id[:8])
```

Comment:
Actually, I think it would make sense to be able to specify a command here (move the logic from `run --local` basically).

The flow would be something like:
- Create Dockerfile
- docker build ...
- repeat until happy:
 * cl docker ... 'python main.py ...'
 * cl docker ... (inside it, run 'pip install')
- Edit Dockerfile to include all the pip install commands
- docker build ...
- docker push ...
- cl run --request-docker-image ...

If we wanted to be slicker, we could have `cl docker` take a `Dockerfile` directly, and have an option for appending commands to the `Dockerfile`.  Thoughts?